query-service/src/integrationTest/java/org/hypertrace/core/query/service/QueryServiceTestUtils.java

```diff
@@ -1,34 +0,0 @@
-package org.hypertrace.core.query.service;
-
-import org.hypertrace.core.query.service.api.ColumnIdentifier;
-import org.hypertrace.core.query.service.api.Expression;
-import org.hypertrace.core.query.service.api.Filter;
-import org.hypertrace.core.query.service.api.LiteralConstant;
-import org.hypertrace.core.query.service.api.Operator;
-import org.hypertrace.core.query.service.api.Value;
-import org.hypertrace.core.query.service.api.ValueType;
-
-public class QueryServiceTestUtils {
```

Comment:
We don't want to put code into the production artifact if it's not being used by production code, this class should remain.

We have 3 of these sets of utils.

- api utils. These are deprecated, we shouldn't be using them or adding to them.
- impl utils. These are the ones used by the runtime code. Tests should use these if available. Add new utils here only if they're used at runtime.
- test utils. These are not used by runtime code, only by tests. If you're adding a util to write a test, put it here.

Reply:
Required methods for create filter and all already exists in `QueryRequestBuilderUtils` in the test package of QueryServiceImpl - `~/hypertrace/query-service/query-service-impl/src/test/java/org/hypertrace/core/query/service/QueryRequestBuilderUtils.java`